plugin/trino-raptor-legacy/src/test/java/io/trino/plugin/raptor/legacy/backup/AbstractTestBackupStore.java

```diff
@@ -28,38 +29,38 @@
 
 public abstract class AbstractTestBackupStore<T extends BackupStore>
 {
-    protected File temporary;
+    protected Path temporary;
     protected T store;
 
     @Test
     public void testBackupStore()
             throws Exception
     {
         // backup first file
-        File file1 = new File(temporary, "file1");
-        Files.write("hello world", file1, UTF_8);
+        File file1 = temporary.resolve("file1").toFile();
+        Files.asCharSink(file1, UTF_8).write("hello world");
```

Comment:
Why did this change?

Reply:
I assume you mean replacing `Files.write` with `Files.asCharSink`? If so, `Files.write` is another deprecated method in Guava that seemed close enough.

Reply:
Ok, fair enough, but put it in a separate commit, since it’s not related to the createTempDir change